sqlmock_test.go

```diff
@@ -354,6 +354,63 @@ func TestPreparedQueryExecutions(t *testing.T) {
 	}
 }
 
+func TestPreparedQueryMultipleExecutions(t *testing.T) {
+	t.Parallel()
+	db, mock, err := New()
+	if err != nil {
+		t.Errorf("an error '%s' was not expected when opening a stub database connection", err)
+	}
+	defer db.Close()
+
+	mock.MatchExpectationsInOrder(false)
```

Comment:
Actually it appears that it _is_ needed, so I think that I'll change the `AllowRepeatedExpectationMatching` function to set the `MatchExpectationsInOrder` rather than requiring users to explicitly set this.